### PR TITLE
MkRepo.java:68-69: Implement languages() method. Don't forget... #953

### PR DESCRIPTION
**Pull request by nhekfqn**

### CODE REVIEW

**Comment by dmarkov** [comment]
Many thanks for the PR, let me find a reviewer for it

**Comment by dmarkov** [comment]
@longtimeago please review this one

File: src/main/java/com/jcabi/github/mock/MkRepo.java [hunk 1]
```diff
@@ -281,7 +279,7 @@ public Notifications notifications() {
 
     @Override
     public Iterable<Language> languages() {
-        throw new NotImplementedException("MkRepo#languages");
+        return new ArrayList<Language>(0);
```
Comment (longtimeago) [inline src/main/java/com/jcabi/github/mock/MkRepo.java]: it would be nice to have several languages inside. Empty list is useless

**Comment by nhekfqn** [comment]
@longtimeago updated, please review.

File: src/test/java/com/jcabi/github/mock/MkRepoTest.java [hunk 2]
```diff
@@ -139,4 +139,20 @@ public void fetchNotifications() throws IOException {
...
+     */
+    @Test
+    public void fetchLanguages() throws IOException {
+        final String user = "testuser4";
+        final Repo repo = new MkRepo(
+                new MkStorage.InFile(),
+                user,
+                new Coordinates.Simple(user, "testrepo4")
+        );
+        MatcherAssert.assertThat(repo.languages(), Matchers.notNullValue());
```
Comment (longtimeago) [inline src/test/java/com/jcabi/github/mock/MkRepoTest.java]: please, also check at least elements count

**Comment by longtimeago** [comment]
@nhekfqn One more remark

**Comment by nhekfqn** [comment]
@longtimeago updated, please review.

File: src/test/java/com/jcabi/github/mock/MkRepoTest.java [hunk 3]
```diff
@@ -139,4 +141,25 @@ public void fetchNotifications() throws IOException {
...
+            new MkStorage.InFile(),
+            user,
+            new Coordinates.Simple(user, "testrepo4")
+        );
+        final Iterable<Language> languages = repo.languages();
+        MatcherAssert.assertThat(languages, Matchers.notNullValue());
+        final int size = 3;
+        MatcherAssert.assertThat(
+            Lists.newArrayList(languages),
+            Matchers.hasSize(size)
```
Comment (longtimeago) [inline src/test/java/com/jcabi/github/mock/MkRepoTest.java]: instead of one-time variable ```size``` please use ```Tv.THREE```

**Comment by nhekfqn** [comment]
@longtimeago updated, please review.

**Comment by longtimeago** [comment]
@nhekfqn Thank you!

**Comment by longtimeago** [comment]
@rultor good to merge

**Comment by yegor256** [comment]
@rultor merge this

**Comment by nhekfqn** [comment]
@rultor @yegor256 I've checked the failing test case, it looks like it is unstable - the issue repoduced once then test passed fine. Please try again to deploy.

Ticket created: https://github.com/jcabi/jcabi-github/issues/1002

**Comment by yegor256** [comment]
@rultor try to merge again

**Comment by dmarkov** [comment]
@longtimeago thank you, added **15 mins** to your acc, payment num is `50842297`; +15 added to your rating, current score is: [+1425](http://www.netbout.com/b/35024?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy now pls

**Pull request merged**